gutenberg-ramp.php

```diff
@@ -39,6 +39,8 @@
  */
 function gutenberg_ramp_load_gutenberg( $criteria = true ) {
 
+	_deprecated_function( 'gutenberg_ramp_load_gutenberg', '0.3', ' built-in filters instead. See https://developer.wordpress.org/reference/hooks/use_block_editor_for_post/ for more information' );
```

Comment:
```suggestion
	_deprecated_function( 'gutenberg_ramp_load_gutenberg', '0.3', 'Use built-in filters instead. See https://developer.wordpress.org/reference/hooks/use_block_editor_for_post/ for more information' );
```

Reply:
I skipped `Use` originally because `_deprecated_function()` will print it:

> PHP Deprecated:  gutenberg_ramp_load_gutenberg is <strong>deprecated</strong> since version 0.3! Use Use built-in filters instead. See https://developer.wordpress.org/reference/hooks/use_block_editor_for_post/ for more information instead. in /var/www/html/wp-includes/functions.php on line 4713

I know the code looks a bit weird, but given this it makes sense.

Reply:
@gudmdharalds that makes sense. We should just add an inline comment to that effect:

```suggestion
	// Ignore the awkward phrasing in the 3rd param; we're skipping the word "Use" because core already adds that.
	_deprecated_function( 'gutenberg_ramp_load_gutenberg', '0.3', ' built-in filters instead. See https://developer.wordpress.org/reference/hooks/use_block_editor_for_post/ for more information' );
```